src/static-build/utils.tsx

```diff
@@ -35,7 +35,9 @@ function addCspAsMetaTag(htmlString: string) {
   // Hash inline scripts from this html file, if there are any
   const scriptHashes = s.hashAllInlineScripts();
   // Generate a strict CSP as a string
-  const strictCsp = StrictCsp.getStrictCsp(scriptHashes, false, true);
+  // enableTrustedTypes: false, enableBrowserFallbacks: true
+  // enableUnsafeEval: true, to accomodate for uses of eval by emscripten. Enabling eval makes the CSP a bit less secure
+  const strictCsp = StrictCsp.getStrictCsp(scriptHashes, false, true, true);
```

Comment:
This is a drive-by: standalone boolean parameters are a bit of an API smell as they are not self-explanatory.

If you can, I’d recommend refactoring it into an options object like so

```js
const strictCsp = StrictCsp.getStrictCsp(scriptHashes, {
  enabledTrustedTypes: false, 
  enableBrowserFallbacks: true, 
  enableUnsafeEval: true
});
```

Reply:
yes thank you, the `options` object is way better. Done! 
(the strict-csp package is also republished with this change)